### ISSUE TITLE
[Feature request] Support yaml anchors

### ISSUE THREAD

**Issue by AndreyNautilus** [rhysd/actionlint#569]
Add support for yaml anchors please. GitHub actions [support yaml anchors](https://github.com/actions/runner/issues/1182) since August 4 2025.

Example (the [workflow](https://github.com/AndreyNautilus/github-actions-playground/blob/main/.github/workflows/yaml-anchors.yaml) and the [run](https://github.com/AndreyNautilus/github-actions-playground/actions/runs/17560228747/job/49874517850)):
```yaml
# this is a valid github workflow. Note 'cmd' yaml anchor
jobs:
  yaml-anchors:
    runs-on: ubuntu-latest
    env:
      SUBJECT: world
    steps:
      - name: Define yaml anchor
        run: &cmd |
          echo "hello, ${SUBJECT}!"
          ls -la
      - name: Use yaml anchor
        env:
          SUBJECT: job
        run: *cmd
```
and this is the error from `actionlint`:
```
.github/workflows/pr-checks.yaml:36:14: expected scalar node for string value but found alias node with "" tag [syntax-check]
   |
36 |         run: *cmd
   |              ^~~~
```

**Comment by muzimuzhi** [comment]
Duplicate of #133.

**Comment by rhysd** [comment]
Closing as duplicate.
